.github/workflows/cluster_endtoend.yml

```diff
@@ -17,30 +17,8 @@ jobs:
     - name: Check out code
       uses: actions/checkout@v2
 
-    - name: Get dependencies
-      run: |
-        sudo apt-get update
-        sudo apt-get install -y mysql-server mysql-client make unzip g++ etcd curl git wget eatmydata
-        sudo service mysql stop
-        sudo service etcd stop
-        sudo ln -s /etc/apparmor.d/usr.sbin.mysqld /etc/apparmor.d/disable/
-        sudo apparmor_parser -R /etc/apparmor.d/usr.sbin.mysqld
-        go mod download
-        wget https://repo.percona.com/apt/percona-release_latest.$(lsb_release -sc)_all.deb
-        sudo apt-get install -y gnupg2
-        sudo dpkg -i percona-release_latest.$(lsb_release -sc)_all.deb
-        sudo apt-get update
-        sudo apt-get install percona-xtrabackup-24
-
-    - name: Installing zookeeper and consul
-      run: |
-        # Only running for shard 18 and 24 where we need to install consul and zookeeper
-        if [[ ${{matrix.name}} == 18 || ${{matrix.name}} == 24 ]]; then
-          make tools
-        fi
-
     - name: sharded cluster_endtoend
       run: |
         source build.env
-        eatmydata -- go run test.go -docker=false -print-log -shard ${{matrix.name}}
+        eatmydata -- go run test.go -print-log -shard ${{matrix.name}}
```

Comment:
this change means that all tests will start running inside docker. Is that what we want?